libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -174,23 +156,8 @@ namespace eosio { namespace chain {
             });
             trx_context.pause_billing_timer();
             IR::Module module;
```

Comment:
one more line here that can get the chop

Reply:
Sorry I should be more thorough. Don't know why my build did not show unused variables warning.